src/metadataparsers/templated/Templated.php

```diff
@@ -29,13 +29,9 @@ public function __construct($settings)
     }
 
     /**
-     * Wrapper mmethod called from within the mik script.
+     * {@inheritdoc}
      *
-     * @param string $record_key
-     *   The current item's record key.
-     *
-     * @return string
-     *   The output of the template.
+     *  Returns the output of the template.
```

Comment:
Because each of these metadata outputs returns slightly different versions of metadata, I went with using `{@inheritdoc}` but including a version specific statement to clarify what you might get back. Normally this was based on the original `@return`